src/ast.ts

```diff
@@ -541,10 +543,25 @@ export function resolveFieldDef(
 
   if (versionInfo.major < 16) {
     const fieldName = fieldNode.name.value;
-    return getFieldDef(compilationContext.schema, parentType, fieldName as any);
+    return (execute as any).getFieldDef(
+      compilationContext.schema,
+      parentType,
+      fieldName as any
+    );
   }
 
-  return getFieldDef(compilationContext.schema, parentType, fieldNode as any);
```

Comment:
Let's add a shim here (https://github.com/zalando-incubator/graphql-jit/blob/main/src/compat.ts) in this repo to work around this change.